students/thorn/lesson03/strformat_lab.py

```diff
@@ -5,15 +5,114 @@
 - String Formatting Lab
 """
 
+import random
+
 def task_one(in_tuple = (2, 123.4567, 10000, 12345.67)):
-    """ Takes a tuple as a parameter and formats its members as follows:
+    """ 
+    Takes a tuple as a parameter and formats its members to the following:
+    'file_002 :   123.46, 1.00e+04, 1.23e+04'
     1: Generates a filename.
     2: Float displayed with 2 decimal places.
     3: Integer displayed in scientific notation with 2 decimal places.
     4: Float displayed in scientific notation to 3 sig figs.
     """
+    new_tuple = "file_{:03d} : {:.02f}, {:.2E}, {:.2E}".format(in_tuple[0], in_tuple[1], in_tuple[2], in_tuple[3])
```

Comment:
you can also use shortcut -> `"".format(*in_tuple)` which will expand each tuple item